storage/cloudspanner/getdb_test.go

```diff
@@ -61,15 +61,22 @@ func GetTestDB(ctx context.Context, t *testing.T) *spanner.Client {
 	}
 }
 
-var dbCount uint32 // Unique per test invocation
+var dbCount struct {
+	sync.Mutex
+	count int
+}
 
 func uniqueDBName(project, instance string) string {
 	// Unique per test binary invocation
 	timestamp := time.Now().UTC().Format("jan-02-15-04-05")
 	testBinary := strings.ToLower(strings.Replace(path.Base(os.Args[0]), ".test", "", 1))
 	invocationID := fmt.Sprintf("%s-%s", timestamp, testBinary)
 
-	database := fmt.Sprintf("%s-%d", invocationID, atomic.AddUint32(&dbCount, 1))
+	dbCount.Lock()
+	dbCount.count++
+	defer dbCount.Unlock()
```

Comment:
nit: typically you'd swap this line and the one above - keep the `Lock()` and `defer Unlock()` right next to each other.

Reply:
done